Changes to linkability and data storage sections (#540)
oauth-wg/oauth-selective-disclosure-jwt: draft-ietf-oauth-selective-disclosure-jwt.md

* Presentation Unlinkability: A Verifier should not be able to link two
  presentations of the same credential.

> * Verifier/Verifier Unlinkability: The presentations made to two different
>   Verifiers should not reveal that the same credential was presented (e.g., if the two
>   Verifiers collude, data leaks from one Verifier to the other, or if both
>   are forced by a third party to reveal the presentations made to them).

Author: ```suggestion
* Verifier/Verifier Unlinkability: The presentations made to two different
  Verifiers should not reveal that the same credential was presented (e.g., if the two
  Verifiers collude, or if they are forced by a third party to reveal the presentations
  made to them, or data leaks from one Verifier to the other).
```

> * Issuer/Verifier Unlinkability (Honest Verifier): An Issuer of a credential

Author: ```suggestion
* Issuer/Verifier Unlinkability (without leak from Verifier): An Issuer of a credential
```

>   should not be able to know that a user presented this credential if
>   the Verifier is not sharing presentation data with the Issuer
>   accidentally, deliberately, or because it is forced to do so.
> * Issuer/Verifier Unlinkability (Colluding/Compromised/Coerced Verifier): An Issuer of a
>   credential should not be able to tell that a user presented this credential to
>   a certain Verifier, even if the Verifier accidentally or deliberately shares
>   presentation data with the Issuer or is forced to do so.

Reviewer: ```suggestion
  should not be able to know that a user presented this credential if
  the Verifier is not accidentally sharing presentation data with the Issuer.
* Issuer/Verifier Unlinkability (Colluding/Compromised/Coerced Verifier): An Issuer of a
  credential should not be able to tell that a user presented this credential to
  a certain Verifier, even if the Verifier deliberately shares
  presentation data with the Issuer or is forced to do so.
```
Colluding/Compromised/Coerced Verifier would only share deliberately and not accidentally, right? likewise, honest verifier would no share deliberately or forced to do so.

Author: Maybe "honest" should be changed to a different name implying "non-leaking"? We could also add "careless" to the Issuer that does leak:
```suggestion
  should not be able to know that a user presented this credential if
  the Verifier is not sharing presentation data with the Issuer
  accidentally, deliberately, or because it is forced to do so.
* Issuer/Verifier Unlinkability (Careless/Colluding/Compromised/Coerced Verifier): An Issuer of a
  credential should not be able to tell that a user presented this credential to
  a certain Verifier, even if the Verifier accidentally or deliberately shares
  presentation data with the Issuer or is forced to do so.
```

>   should not be able to know that a user presented this credential if
>   the Verifier is not sharing presentation data with the Issuer
>   accidentally, deliberately, or because it is forced to do so.
> * Issuer/Verifier Unlinkability (Colluding/Compromised/Coerced Verifier): An Issuer of a
>   credential should not be able to tell that a user presented this credential to
>   a certain Verifier, even if the Verifier accidentally or deliberately shares
>   presentation data with the Issuer or is forced to do so.

Reviewer: ```suggestion
  should not be able to know that a user presented this credential unless
  the Verifier is sharing presentation data with the Issuer
  accidentally, deliberately, or because it is forced to do so.
* Issuer/Verifier Unlinkability (Colluding/Compromised/Coerced Verifier): An Issuer of a
  credential should under no circumstances be able to tell that a user presented this credential to
  a certain Verifier. In particular this includes cases when the Verifier accidentally or deliberately shares
  presentation data with the Issuer or is forced to do so.
```

In all cases, unlinkability is limited to cases where the disclosed claims do
not contain information that directly or indirectly identifies the user. For
example, when a taxpayer identification number is contained in the disclosed claims, the Issuer and
Verifier can easily link the user's transactions. However, when the user only
discloses a birthdate to one Verifier and a postal code to another Verifier, the two Verifiers should not be able to determine that they were interacting with the same user.

> Issuer/Verifier unlinkability with a colluding, compromised or coerced Verifier cannot be

Reviewer 2: Oxford?
```suggestion
Issuer/Verifier unlinkability with a colluding, compromised, or coerced Verifier cannot be
```

> Issuer/Verifier unlinkability with a colluding, compromised, or coerced Verifier cannot be

Author: ```suggestion
Issuer/Verifier unlinkability with a careless, colluding, compromised, or coerced Verifier cannot be
```
